src/test/miner_tests.cpp

```diff
@@ -126,35 +126,40 @@ BOOST_AUTO_TEST_CASE(CreateNewBlock_validity)
     BOOST_CHECK(pblocktemplate = CreateNewBlock(chainparams, scriptPubKey));
     delete pblocktemplate;
 
+    const CAmount BLOCKSUBSIDY = 500*COIN;
+    const CAmount LOWFEE = 10*CENT;
```

Comment:
Why bumping fees 10x here?

Reply:
Bumped this because I also had to bump BLOCKSUBSIDY. Just checked execution of the tests and confirmed that this was not needed. Added a commit on top to remove the bumping.